src/service.py

```diff
@@ -1,118 +1,176 @@
-import sys
+import os
+from pathlib import Path
+
 import tmt
 import logging
-from pathlib import Path
-from src import html_generator as html
 from src.utils import git_handler as utils
```

Comment:
Just noticed this weird import. Adding note even though it should be fixed as part of refactoring to https://github.com/teemtee/tmt/pull/2983